playbook.yml

```
@@ -93,59 +93,12 @@
       state: touch
       mode: '0644'
 
-  - name: Remove EnvironmentFile
-    ansible.builtin.file:
-      path: ~/storage/nextcloud/nextcloud.env
-      state: absent
-
-  - name: Touch EnvironmentFile
-    ansible.builtin.file:
-      path: ~/storage/nextcloud/nextcloud.env
-      state: touch
+  - name: Create EnvironmentFile
+    ansible.builtin.template:
+      src: nextcloud.env.j2
```

```suggestion
      src: templates/nextcloud.env.j2
```
Or is the `templates` dir the lookup dir anyway?

---

Yes, `templates` is the default location, that is searched first.

https://docs.ansible.com/ansible/latest/playbook_guide/playbook_pathing.html